common/lib/xmodule/xmodule/templates/problem/customgrader.yaml

```diff
@@ -4,11 +4,31 @@ metadata:
     markdown: !!null
 data: |
       <problem>
+      <p>    
+      In custom Python-evaluated input (also called “write-your-own-grader”
+      problems), the grader uses a Python script that you create and embed in
+      the problem to evaluate a learner’s response or provide hints. These
+      problems can be any type. Numerical input and text input problems are
+      the most common write-your-own-grader problems.
+      </p>
+      <p>
+      You can use script tag format or answer tag format to create these problems. 
+      </p>
       <p>
-      A custom python-evaluated input problem accepts one or more lines of text input from the
-      student, and evaluates the inputs for correctness based on evaluation using a 
-      python script embedded within the problem.
+      You can create custom Python-evaluated input problems that provide
+      partial credit or that randomize variables in the Python code. You can
+      also add images to the solution by using an HTML "img" tag. Note that
+      the "img" tag must be between the "div" tags that are inside the
+      "solution" tags, and that learners do not see these images until they
+      click the "Show Answer" button.      
       </p>
+      <p> For more information, see <a
+      href="http://edx.readthedocs.org/projects/edx-partner-course-
+      staff/en/latest/exercises_tools/custom_python.html">Write-Your-Own-
+      Grader Problem</a> in <i>Building and Running an edX Course</i>.
+      </p>      
```

Comment:
I decided to hard-code these links in the templates for course authors' convenience. If we ever move these docs we'll have to update the links, but we should be making sure we don't break anything any time we move the docs, so hopefully we'll remember. (And if we don't, I'm sure a course team will be very quick to point it out...)